src/compiler/filesystem_local_win.cc

```diff
@@ -29,8 +29,15 @@
 namespace toit {
 namespace compiler {
 
+// We need to pick between '\' and '/'. As much as I hate it,
```

Comment:
Is this only used by `FilesystemLocal::path_separator` now? Maybe move the constant and comment there.